### DISCUSSION TITLE
Title attribute should include localization

### DISCUSSION THREAD

**Discussion by scrothers** [livewire/livewire#7993] [3. Feature ideas]
The title attribute should include the ability to select a localized key.

Pretty straight forward, I just came across the issue with attributes not being able to use __() today, I'm new to attributes and I didn't realize it wouldn't work as I expected.

I'll try and work up a PR this weekend for it, but if someone knows how to do this and can just slap it up, that would also be great (and a learning experience) for me.

**Comment by joshhanley** [comment]
@scrothers yeah makes sense to me. Although you could just wrap `$title` in the translation helper inside your layout file. But we do already support translation for the `#[Validation]` attributes so don't see why the `#[Title]` attribute couldn't be the same.
